src/app/space/forge-wizard/components/pipeline-view/pipeline-view.component.html

```diff
@@ -0,0 +1,42 @@
+<ul class="selection-list">
+  <li title="{{choice.name}}" [class.selected]="field.value === choice.id" (click)="field.value=choice.id;"
+      *ngFor="let choice of field.valueChoices">
+    <input class="selection-list-input user-select-none form-control" title="choice.name"
```

Comment:
Something is causing a box-shadow on radio buttons in step 2a of the wizard; I think it's coming from this .form-control